apps/evisa/behaviours/email-customer.js

```diff
@@ -15,11 +15,17 @@ module.exports = emailConfig => superclass => class EmailCustomer extends superc
   async successHandler(req, res, next) {
     if (req.body['continue-button']) {
       let emailAddress = req.sessionModel.get(SESSION.EMAIL_FIELD);
-      if (emailAddress) {
-        let personalisation = this.getCustomerPersonalisation(req.sessionModel);
-        this.emailer.sendCustomerEmail(emailAddress, personalisation);
+      emailAddress = '';
```

Comment:
Is this left in from a test? seems like it would assign empty string to any and every email coming through

Reply:
Yes that's pure debug, I'll remove it